server/itx/server_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019 IoTeX
```

Comment:
I feel these tests are not so useful, as it's a integration place. Instead, we need to write more e2e integration tests